config/envs/default.py

```diff
@@ -45,3 +45,100 @@ class DefaultConfig:
     FUND_ENDPOINT = "/funds/{fund_id}"
     FUND_ROUNDS_ENDPOINT = "/funds/{fund_id}/rounds"
     FUND_ROUND_ENDPOINT = "/funds/{fund_id}/rounds/{round_id}"
+
+    SQLALCHEMY_DATABASE_URI = environ.get("DATABASE_URL")
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+    # COF FORMS BASE CONFIG
+    COF_R2_FORMS = [
+        {
+            "form_minting_name": "applicant-information",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "asset-information",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "community-benefits",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "community-engagement",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "community-representation",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "community-use",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "declarations",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "environmental-sustainability",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "feasibility",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "funding-required",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "inclusiveness-and-intergration",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "local-support",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "organisation-information",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "project-costs",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "project-information",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "project-qualification",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "risk",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "skills-and-resources",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "value-to-the-community",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "upload-business-plan",
+            "questions": [],
+        },
+    ]
+
+    FUND_ROUND_FORMS = {
```

Comment:
As these all seem to be test data should they come out of default and go in the relevant envs so they don't end up in prod?